controllers/solrbackup_controller.go

```diff
@@ -146,7 +145,7 @@ func (r *SolrBackupReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 	}
 
 	// Schedule the next backupTime, if it doesn't have a next scheduled time, it has recurrence and the current backup is finished
-	if backup.Status.IndividualSolrBackupStatus.Finished {
+	if backup.Status.IndividualSolrBackupStatus.Finished && backup.Spec.Recurrence.IsEnabled() {
```

Comment:
[+1] Weird that the comment on L147 mentions recurrence-being-enabled as being a prereq, but we never checked for it before now.  Must've just been an oversight on my part?

Anyway, good catch!